internal/client.go

```diff
@@ -213,61 +224,113 @@ func (client *Client) setupUI() {
 	client.app.EnableMouse(true)
 }
 
-func (client *Client) shortenPlotId(id string) string {
+func shortenPlotId(id string) string {
 	if len(id) < 20 {
 		return ""
 	}
 	return fmt.Sprintf("%s...%s", id[:10], id[len(id)-10:])
 }
 
+// Active plots
```

Comment:
The goal is to group most of the code for each table in to its own section.  It might make sense to make a function `setupActivePlotsTable()` as well, which pulls out the code from `setupUI()`.